modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/openapi/v1_0/ObjectEntryOpenAPIResourceImpl.java

```diff
@@ -103,14 +107,65 @@ public Map<String, Field> getFields(UriInfo uriInfo) throws Exception {
 
 		Map<String, Schema> properties = schema.getProperties();
 
+		List<ObjectRelationship> objectRelationships =
+			_objectRelationshipLocalService.
+				getObjectRelationshipsByObjectDefinitionId2(
+					_objectDefinition.getObjectDefinitionId());
```

Comment:
I think we could be iterating over all the relationships where the object definition 2 matches with the current one (that is one to many and many to many)